packages/IMAPClient-Core.package/ICAuthenticationInfo.class/instance/usernameAsIMAPString.st

```diff
@@ -0,0 +1,4 @@
+accessing
+usernameAsIMAPString
+
+	^ '"', self username, '"'
```

Comment:
Could there be an issue here with `"` characters in the username itself? Same goes for the password...